src/admin/config.njk

```diff
@@ -0,0 +1,86 @@
+---
+permalink: /admin/config.yml
+---
+backend:
+  name: git-gateway
+  branch: main
+media_folder: "src/assets/uploads"
+public_folder: "/assets/uploads"
+
+# Run "npx netlify-cms-proxy-server" for local backend
+local_backend: true
+editor:
+  preview: false
+i18n:
+  structure: multiple_folders
+  locales: [{% for language in config.languages %}{{language.code}}{% if loop.last !== true %}, {% endif %}{% endfor %}]
+  default_locale: {{ config.languages[0].code }}
```

Comment:
It appears that you are specifying the default language based on the first item in the languages array. However, in feed.njk it appears to explicitly look for "en-CA". It would probably be better to have the default language set in the configuration and just check for that wherever you need to make some action based on the default locale.

Reply:
@BlueSlug and I discussed this and felt that the first language should be the default rather than adding another config item for the default language. I should update the feed to use the first language as default, though.

Reply:
I'm on the fence now, actually. A separate config item would mean a bit of duplication, though it would make it fairly explicit which is the default which *could* be easier for an author to fiddle with.

Reply:
The problem is that Netlify CMS can't dynamically update the list of languages that a user has added to allow one to be selected as default. SO if you need to add a new language and then change the default language, you'd need to:

1. Add the new language.
2. Publish.
3. Wait for the site to build.
4. Reload the CMS.
5. Select the new default language from the list.
6. Publish.

I do think that having the first language be the default makes sense and is less convoluted. Perhaps if we add a hint to the CMS which says "the first language in this list will be the site's default language— reorder the list if you want to make the default a different language".

Reply:
I'm a bit confused. Can you add new languages and reorder the list of them through the CMS directly? What is the difference that allows reordering languages but not selecting a default when adding a new language?